doc/source/index.rst

```diff
@@ -3,19 +3,19 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+=================================================
 Welcome to MDAnalysis User Guide's documentation!
 =================================================
 
 **MDAnalysis version:** |MDAnalysis_version|
 
 **Last updated:** |today|
 
-**MDAnalysis** (`www.mdanalysis.org`_) is an object-oriented python
-toolkit to analyze molecular dynamics files and trajectories generated by
-CHARMM_, Gromacs_, Amber_, NAMD_, LAMMPS_, `DL_POLY`_ and other
-packages (see :ref:`formats` for full information). MDAnalysis can write
-most of these formats, too, together with atom selections for use in
-Gromacs_, CHARMM_, VMD_ and PyMol_ (see :ref:`selection-exporters`). It uses Numpy_ arrays to provide a flexible and relatively fast framework for complex analysis tasks. 
+**MDAnalysis** (`www.mdanalysis.org`_) is a Python
```

Comment:
I like the updated blurb! 

We should come up with a consistent set of "high level" paragraphs such as this one, the one for https://numfocus.org/project/mdanalysis and for GSoC.